spec/test_track_spec.rb

```diff
@@ -65,16 +65,4 @@
       end
     end
   end
-
-  describe ".fully_qualified_cookie_domain_enabled?" do
```

Comment:
Doh! I didn't see them in the changes so I assumed they were missing. Sort of funny considering I'm the one who wrote them. #facepalm